atomic_reactor/plugins/exit_koji_tag_build.py

```diff
@@ -77,11 +76,8 @@ def run(self):
 
         build_id = self.workflow.exit_results.get(KojiImportPlugin.key)
         if not build_id:
-            build_id = self.workflow.exit_results.get(KojiPromotePlugin.key)
-            if not build_id:
-                self.log.info('No koji build from %s or %s', KojiImportPlugin.key,
-                              KojiPromotePlugin.key)
-                return
+            self.log.info('No koji build from %s', KojiImportPlugin.key)
+            return
```

Comment:
Looks like this block is not covered by unit testing.

Reply:
Fixed.